homeassistant/components/laundrify/__init__.py

```diff
@@ -0,0 +1,61 @@
+"""The laundrify integration."""
```

Comment:
What you could try is to add your component to `.strict-typing`. When you run `mypy homeassistant/components/laundrify`, it will show you how close you are to strict typing. Perhaps the amount of fixes is relatively small, and it would be great to have a new integration strictly typed from the beginning.

Reply:
I added the component to `.strict-typing` and fixed the type hints in https://github.com/home-assistant/core/pull/65090/commits/1b179a03faaf20e528d1e0a4433e768c1e8717eb

`mypy` doesn't return any errors, hope I didn't miss anything :)

```bash
root ➜ /workspaces/core (laundrify ✗) $ mypy homeassistant/components/laundrify
Success: no issues found in 6 source files
```